### PR TITLE
Add scil_compute_streamlines_dice.py script

### PR DESCRIPTION
**Pull request by cbedetti**
Implementation of `scil_compute_streamlines_weighted_dice.py` from bitbucket scilpy.
I removed weighted from the name since it's an option.

It works with two trks of mine.

Is there any testing data?

### CODE REVIEW

File: scripts/scil_compute_streamlines_dice.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(
+        parser, args.in_bundle1, optional=args.reference)
+    assert_inputs_exist(
+        parser, args.in_bundle2, optional=args.reference)
```
Comment (arnaudbore) [inline scripts/scil_compute_streamlines_dice.py]: These four lines should be merged into one assert_inputs_exist call

**Comment by arnaudbore** [comment]
You can provide your own data, but you're right we should provide a dataset to run some tests.

**Comment by frheault** [comment]
There is code in PR #67 computing the Dice and WDice, I think it should be the same function in a module.
You could steal some of that code and I will deal with the conflict when PR #67 is ready to merge

File: scripts/scil_compute_streamlines_dice.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+
+"""
+Compute the dice coefficient between two streamline files.
```
Comment (boiteaclou) [inline scripts/scil_compute_streamlines_dice.py]: Would it be better to change streamline here by tractogram ?

File: scripts/scil_compute_streamlines_dice.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                        help="Tractogram filename. Format must be one of \n"
+                        "trk, tck, vtk, fib, dpy.")
+
+    add_reference_arg(parser)
+
+    parser.add_argument("--weighted", action="store_true",
+                        help="Weight the dice coefficient based on the \n"
+                        "number of tracts passing through a voxel")
+
+    parser.add_argument("--indent", type=int, default=2,
```
Comment (boiteaclou) [inline scripts/scil_compute_streamlines_dice.py]: By default, do we pretty print with 2 or 4 indent ?

**Comment by cbedetti** [comment]
Reading PR #67 of @frheault , there's already all it needs.
I think this PR should be closed.

**Pull request closed without merging**